update_bench_traces_rel_assert.py

```diff
@@ -9,13 +9,20 @@
 from gobexec.model.tools import ExtractTool
 from gobexec.output.renderer import FileRenderer, ConsoleRenderer, MultiRenderer
 
+goblint_assert = GoblintTool(
+    name="goblint_assert",
+    program=str(Path("../analyzer/goblint").absolute()),
+    args=["-v", "--conf", str(Path("../analyzer/conf/traces-rel.json").absolute()), "--enable", "dbg.debug","--set", "trans.activated[+]", "assert"],
+    dump = "assert"
+)
 
 def index_tool_factory(name, args):
     goblint = GoblintTool(
         name=name,
         program=str(Path("../analyzer/goblint").absolute()),
         args=["-v", "--conf", str(Path("../analyzer/conf/traces-rel.json").absolute()), "--enable", "dbg.debug"] + args,

```

Comment:
Since the assertion transformation now outputs SV-COMP things with `__VERIFIER_*`, this should also have `--enable ana.sv-comp.functions` I think.